tests/lib/rules/eol-last.js

```diff
@@ -54,6 +54,24 @@ ruleTester.run("eol-last", rule, {
             options: ["windows"],
             errors: [{ message: "Newline required at end of file but not found.", type: "Program" }],
             output: "var a = 123;\r\n   \r\n"
+        },
```

Comment:
Thanks for adding these! Could you add a couple of new tests for the "valid" scenarios as well?